apps/saleor-app-checkout/test-utils.ts

```diff
@@ -89,7 +91,7 @@ const tryParse = (text: string | undefined) => {
 
 export const setupPollyMiddleware = (server: PollyServer) => {
   // Hide sensitive data in headers or in body
-  server.any().on("beforePersist", (_, recording, event) => {
+  server.any().on("beforePersist", (_, recording, _event) => {
```

Comment:
can we just remove it?
```suggestion
  server.any().on("beforePersist", (_, recording) => {
```